### PR TITLE
Here the defaults for map creation have been adjusted to disable thos…

### PR DESCRIPTION
**Pull request by siawoosh**
…e functionalities which were specifically (optimized) for humans.

Hi Evelyne, Karsten, and Enrico,

as a response to Graham and Phillips(?) problem with processing mouse data, I created a new default file in the local defaults which is adjusted to processing mouse or postmortem data (assuming there was not inter-scan motion). 

Siawoosh

### CODE REVIEW

**Comment by ktabelow** [comment]
Hi Siawoosh, 

why does this create conflicts?

Should we dig into it?

Best,

Karsten

File: config/hmri_defaults.m [hunk 1]
```diff
@@ -412,4 +411,24 @@
 % Number of Gaussians per tissue class
 hmri_def.proc.nGauss = [2 2 2 3 4 2]; % originally in SPM [1 1 2 3 4 2]
 
+%==========================================================================
+% Experimental features 
+%==========================================================================
+hmri_def.errormaps  = true;
```
Comment (halirutan) [inline config/hmri_defaults.m:417]: What did you do here?

Comment (siawoosh) [reply config/hmri_defaults.m:417]: It allows you to switch on/off creation of hte error maps.

**Comment by halirutan** [comment]
We reviewed it with @siawoosh, @mfcallaghan, @DerOrfa, @lukeje, @antoinelutti. Does not apply anymore.

File: config/hmri_defaults.m (hunk 1 above)
Comment (halirutan) [reply config/hmri_defaults.m:417]: :joy: Nice.

**Pull request closed without merging**